t/sanity.t

```diff
@@ -315,7 +315,7 @@ install: all
 	cp -r $OPENRESTY_BUILD_DIR/pod $(DESTDIR)/usr/local/openresty/
 	cd $OPENRESTY_BUILD_DIR/nginx-1.11.2 && $(MAKE) install DESTDIR=$(DESTDIR)
 	mkdir -p $(DESTDIR)/usr/local/openresty/site/lualib $(DESTDIR)/usr/local/openresty/site/pod $(DESTDIR)/usr/local/openresty/site/manifest
-	ln -sf /usr/local/openresty/nginx/sbin/nginx $(DESTDIR)/usr/local/openresty/bin/openresty
+	ln -sf $(DESTDIR)/usr/local/openresty/nginx/sbin/nginx $(DESTDIR)/usr/local/openresty/bin/openresty
```

Comment:
Ah! yes, our internal packages are not like the official linux packages. Our packages are installed on non-standard locations, an all the executables are located on the same directory. I will change it to not break things, and somehow make our build system do what we need...